src/pl/devsmentoring/java/Module5/Exercise20.java

```diff
@@ -4,42 +4,49 @@
 
 public class Exercise20 {
     public static void main(String[] args) {
-        // 'zielony', 'czerwony', 'niebieski', 'czarny', 'fioletowy', 'granatowy', 'niebieski', 'czarny',
-        // 'czarny', 'zielony', 'cytrynowy', 'granatowy', 'niebieski', 'indygo', 'zielony', 'czerwony'
-
 
         List<String> colourList = new ArrayList<>(Arrays.asList("zielony", "czerwony", "niebieski", "czarny",
                 "fioletowy", "granatowy", "niebieski",
                 "czarny", "czarny", "zielony", "cytrynowy", "granatowy",
                 "niebieski", "indygo", "zielony", "czerwony"));
 
+        int greenCounter = Collections.frequency(colourList, "zielony");
+        int redCounter = Collections.frequency(colourList, "czerwony");
+        int blueCounter = Collections.frequency(colourList, "niebieski");
+        int blackCounter = Collections.frequency(colourList, "czarny");
+        int purpleCounter = Collections.frequency(colourList, "fioletowy");
+        int navyCounter = Collections.frequency(colourList, "granatowy");
+        int lemonCounter = Collections.frequency(colourList, "cytrynowy");
+        int indygoCounter = Collections.frequency(colourList, "indygo");
```

Comment:
jakbyśmy dodali przykładowo kolor żółty to musielibyśmy znów przerabiać pod to kod. 

Podpowiedź: Zrób mapę gdzie kluczem będzie kolor (czyli String), a wartością Integer (czyli ilość). Loopując przez listę dodajesz kolory do mapy, jeśli taki kolor już istnieje - iterujesz wartość o 1